README.md

```diff
@@ -67,8 +67,15 @@ chmod u+x git-clang-format
 
 Note: installation of the script will be added to build of AliceO2 software stack.
 
-##### Checking formatting
-Now, `git clang-format` will invoke `clang-format` on the changes in current files
+##### Install formatting configuration
+```
+cd <O2 source directory>
+curl -O https://raw.githubusercontent.com/AliceO2Group/CodingGuidelines/master/.clang-format
```

Comment:
Is cmake doing this automatically? If not I think it is imperative to do so (otherwise consequences can be dramatic for instance if another .clang-format file is found further up the directory tree).

Reply:
I could do it in cmake indeed. good idea, let me do that.